README: New chapter about "Location Sponsor / Location and Company Briefing" (#30)
WebEngDUS/WebEngDUS: README.md

Everything you need to know about how we run the *Web Engineering DUS Meetup*.

## Location Sponsor / Location and Company Briefing

The *Web Engineering DUS Meetup* depends on the support of local companies.

> Local companies can sponsor the meetup by providing us with a location (e.g., a room), technical equipment, and some food and drinks to run the meetup.

Reviewer: ```suggestion
Local companies can sponsor the meetup by providing us with a location, technical equipment, and some food and drinks during the meetup.
```
